build/travis/prepare.sh

```diff
@@ -9,6 +9,16 @@ main() {
     skeleton="${TRAVIS_BUILD_DIR}/resources/test-skel"
 
     sudo cp "${svc}" /etc/systemd/system/ && sudo systemctl daemon-reload
+
+    sudo ls -lAh /home/travis/.config
+    if [[ ! -d /home/travis/.config/git ]]; then
+        mkdir -p /home/travis/.config/git
+        touch /home/travis/.config/git/ignore
+        touch /home/travis/.config/git/attributes
+    fi
+    sudo ls -lah /home/travis/.config/git
+    sudo chmod -R 755 /home/travis/.config/git
+    sudo ls -lah /home/travis/.config/git
```

Comment:
None of this is making any difference, probably best just add `-H` to the sudo'd git commands and create a dummy home folder within `storage/www-home` or something that contains a basic `.config/git/` structure.
```suggestion
```